modules/manage/pages/manage-nodes/manage-data-service-and-rebalance.adoc

```diff
@@ -0,0 +1,32 @@
+= Manage a Data Service and Rebalance
+:description: pass:q[Add or remove Data Service on existing nodes in a cluster and rebalance the cluster.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+Add or remove a Data Service on existing nodes in a cluster from the Couchbase UI, REST API, or CLI.
+
+== Add a Data Service on an Existing Node
+
+To add a Data Service on an existing node of a cluster from the UI, REST API, or CLI:
+
+. Remove the node that you want to add the Data Service to using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with enabling the Data Service.
+
+Alternatively, you can add a new node with the necessary services configuration, including the Data Service, using xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance].
+Then remove the extraneous node that you no longer need using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+
+== Remove a Data Service from an Existing Node
+
+To remove a Data Service from an existing node of a cluster using the UI, REST API, or CLI:
+
+. Remove the node with the Data Service that you want to remove using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with disabling the Data Service.
+
+NOTE: During the node removal operation, if there are other services on the node that you are removing for the Data Service reconfiguration, and you do not want those services to be impacted when the node is removed from the cluster, you may need to add those services to another node temporarily.
+Then add a new node temporarily with those services enabled.
```

Comment:
Instead of Then, it should be an Or.

Or, add a new node temporarily with the impacted services enabled.